### ISSUE TITLE
“Jump to non-overlayed popups” links

### ISSUE THREAD

**Issue by yellowled** [s9y/Serendipity#290]
Quoting vom #234:

> I tried it, entering categories, tags for example. I first thought it to be not working, then I discovered the "toolbar" like top buttons have to be clicked 2 times to jump to the fields and the inner-form jump for shown categories and tags to be disabled at all. I think that needs improvement to be more intuitive.

**Comment by yellowled** [comment]
@ophian Did you experience the 2-click issue in a specific browser? Because it works fine with a “normal” click in Chrome for me …

**Comment by yellowled** [comment]
Unable to reproduce this in Chrome 40 or Firefox 35 on OS X.

**Comment by ophian** [comment]
Huh - It was related to the `Enable use of popup windows for the backend YES` option. Fine without.
Shall we close it and open a new one? :)
And strange!
Enable use of popup windows for the backend `No`
Force specific backend popups: `categories, tags`
Open an entry with or without set categories or tags, click the in-form field, which now jumps to the overay, same as the "toolbar" cat and tags buttons. Am I not doing this right?

**Comment by yellowled** [comment]
> It was related to the Enable use of popup windows for the backend YES option. Fine without.

It works fine for me with those settings as well.

> Open an entry with or without set categories or tags, click the in-form field, which now jumps to the overay, same as the "toolbar" cat and tags buttons.

I'm not sure what you mean by “click the in-form field”. Do you mean the (in that case) “No categories” label? That does not do anything for me (which is probably something to fix).

Also, I just noticed that if overlays are disabled for categories, the categories section in entry meta data displays the button to toggle the cat view, which it shouldn't. That's definitely something I need to fix.

**Comment by ophian** [comment]
Its really strange, since I now have 2 local dev blogs behaving differently. One has `categories, tags` the other only `tags` in the granular option. The first one opens all the jumps into an overlay, the second is doing well, but still relies on this 2-click issue (latest FF build on WOS). (May the latter perhaps have to do with cookie or other collabsible box remember settings?)
And it is a pity I can't check nbe any more. :(

> I'm not sure what you mean by “click the in-form field”. Do you mean the (in that case) “No categories” label?

Yes, and its title, both jump normally.

**Comment by yellowled** [comment]
> latest FF build on WOS

What does that mean? Is it the stable FF (which currently is 35.0.1) or is it the FF dev edition (a.k.a. Aurora)? Because the latter is probably buggy (and not a browser that I can test).

> May the latter perhaps have to do with cookie or other collabsible box remember settings?

I don't think so, but I really need to test this more in-depth, which I don't have time for at the moment (and it takes time because there's a lot of possibilities to include).

**Comment by ophian** [comment]
> Because the latter is probably buggy

I should know that, shouldn't I?! 
No  its 35.0.1. But I don't think it was a matter of my browsers since Chrome had the same (see above). I could use our new synced dev blog, if it had something to test with already...

**Comment by yellowled** [comment]
> I could use our new synced dev blog, if it had something to test with already...

Well, you have a login for that. I'm sure it's fine to add some test content there.

**Comment by yellowled** [comment]
@ophian I did some tests and now have a theory for this. Did you maybe have “Erweiterte Optionen” *expanded* when you tested this? Because in that case it will indeed take 2 clicks to jump to categories or tags because the first click will *collapse* the (expanded) “Erweiterte Optionen”. The JS for the toggle buttons needs to be more complicated to consider that possibility.

**Comment by ophian** [comment]
That is what I suggested before. Nav-Button tag opens the collabsible box to fall directly to tag "fieldset". Then this box is open and every new click will need 1 additional click to check an already opened box before.

**Comment by yellowled** [comment]
Sorry, but I did not get that from

> May the latter perhaps have to do with cookie or other collabsible box remember settings?

because technically, it is not so much related to a cookie or settings. It is related to the fact that the JS for collapsing/expanding these sections does not check whether a section is collaped or expanded. I remember having this on my agenda even before we introduced the granular settings for this. Apparently, I never got around to implementing it.
